google/cloud/datastore/aggregation.py

```diff
@@ -204,7 +205,7 @@ def fetch(
             >>> client.put_multi([andy, sally, bobby])
             >>> query = client.query(kind='Andy')
             >>> aggregation_query = client.aggregation_query(query)
-            >>> result = aggregation_query.count(alias="total").fetch()
+            >>> result = aggregation_query.count(alias="total").fetch(fetch=5)
```

Comment:
I think this should say `limit=5`?

Reply:
Ah yes, I will fix in the next commit. Still need to increase the test coverage.